tsl/test/expected/continuous_aggs_policy.out

```diff
@@ -0,0 +1,97 @@
+-- This file and its contents are licensed under the Timescale License.
```

Comment:
What is the separation between this test and `continuous_aggs_bgw`? Or, put differently, why aren't they merged into one?

Reply:
continuous_aggs_bgw checks if the execution jobs run as expected  and need to eb plugged into the bgw test  framework. This test is specific to policy validation.